docs/source/system_reference_guide/faq/request_data.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>The purpose of this form is to provide a way for users to request data to be ingested into the <a href="https://stac-browser.maap-project.org/" target="_blank">MAAP STAC Catalog</a> for use.</blockquote><p> </p><p>A question because it's still not 100% clear to me. Is this form encompassing both requests for (1) data storage (having the data in the MAAP S3 bucket) and (2) MAAP STAC metadata storage ? I was under the impression that no, this form is just for (1) and we want another one for (2). </p><p> </p><p>If we want these processes to happen <em>together </em>though,  we could use only this form, but maybe it has to be enriched (we can do that later) with information we will need to create the metadata. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
cc @smk0033 @wildintellect

Reply:
Ohh yeah, I didn't think about S3, my mind automatically went to STAC for some reason. I'll wait for Alex to answer on that one

Reply:
This form is for "official" import to MAAP, aka STAC which implies it's already on S3 or a copy will be placed there.

Reply:
Yes, eventually we'll need more info from users. This was step one. You can see what details to add is still being debated https://github.com/MAAP-Project/Community/pull/712